lib/smart_answer_flows/can-i-get-a-british-passport.rb

```diff
@@ -1,75 +0,0 @@
-status :draft
-satisfies_need "100983"
```

Comment:
This makes me think: do they need too update anything in Maslow (user needs tracker)? Mind checking with Gwen?